src/managementgroup/azext_managementgroup/generated/_params.py

```diff
@@ -0,0 +1,166 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    get_three_state_flag,
+    get_enum_type
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('managementgroup management-group list') as c:
+        c.argument('skiptoken', type=str, help='Page continuation token is only used if a previous operation returned '
+                   'a partial result.  If a previous response contains a nextLink element, the value of the nextLink '
+                   'element will include a token parameter that specifies a starting point to use for subsequent '
+                   'calls.')
+
+    with self.argument_context('managementgroup management-group show') as c:
+        c.argument('group_id', type=str, help='Management Group ID.')
+        c.argument('expand', arg_type=get_enum_type(['children', 'path', 'ancestors']), help='The $expand=children '
+                   'query string parameter allows clients to request inclusion of children in the response payload.  '
+                   '$expand=path includes the path from the root group to the current group.  $expand=ancestors '
+                   'includes the ancestor Ids of the current group.')
```

Comment:
Please refine the help message, because It seems to illustrate the feature of this parameter from the perspective of REST API rather than from the perspective of command parameters.
Please explain the parameters from the perspective of command parameters, otherwise users will be confused.